scripts/package-json-minify.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+/**
+ * This is a transform stream used to minify "package.json" results
```

Comment:
please rename this script.  when we think minification, we think uglification... I'm not sure what it should be called (distillation?) but it's a misnomer as is.

Reply:
Other possibilities:
* quantizify
* remandraify
* vestigify
* basilify
* cuttotheboneify

Reply:
I apologize--I shouldn't have used "we" in my original comment, since you and I are in disagreement.

But I can *also* find a source documenting my definition if we want to take the pedantry to another level.

*I* would expect "minification" of a JSON to be "not prettified"; in other words, the original JSON less any formatting.